modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -10,52 +20,100 @@ If you do not need to review the JSON files that the `ccoctl` tool creates befor
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this location.
 ====
 
 .Prerequisites
 
 * Extract and prepare the `ccoctl` binary.
+ifdef::alibaba[]
+* Obtain the AccessKey (AK) of the RAM user you wish to use.
+* Use the AK of the chosen RAM user to configure the {alibaba} SDK client's credential provider chain for link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#1-environment-credentials[Environment Credentials] mode or through link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#2-credentials-file[Credentials File] mode.
+//Note: those Alibaba Cloud docs links are officially intended for user consumption (https://github.com/openshift/cloud-credential-operator/pull/412/files#r764509590). We would prefer not to link to GitHub like this, but an alternative has not yet presented itself.
+endif::alibaba[]
 
 .Procedure
 
 . Extract the list of `CredentialsRequest` objects from the {product-title} release image:
 +
 [source,terminal,subs="+quotes"]
+ifdef::aws[]
+----
+$ oc adm release extract --credentials-requests --cloud=aws --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
 ----
-$ oc adm release extract --credentials-requests --cloud=aws --to=__<path_to_directory_with_list_of_credentials_requests>__/credrequests quay.io/__<path_to>__/ocp-release:__<version>__
+endif::aws[]
+ifdef::alibaba[]
 ----
+$ oc adm release extract --credentials-requests --cloud=alibabacloud --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+----
+endif::alibaba[]
 
 . Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory:
 +
 [source,terminal,subs="+quotes"]
+ifdef::aws[]
+----
+$ ccoctl aws create-all --name=<name> --region=<aws_region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests
 ----
-$ ccoctl aws create-all --name=__<name>__ --region=__<aws_region>__ --credentials-requests-dir=__<path_to_directory_with_list_of_credentials_requests>__/credrequests
+endif::aws[]
+ifdef::alibaba[]
 ----
+$ ccoctl alibabacloud create-ram-users --name <name> --region=<alibaba-region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests
+----
++
+where:
++
+** `<name>` is the name used to tag any cloud resources that are created for tracking.
+** `<alibaba-region>` is the Alibaba Cloud region in which cloud resources will be created.
+** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files for the component `CredentialsRequest` objects.
```

Comment:
I do not see `where:` part explaining individual parameters for aws

Reply:
The way this is included in the AWS STS docs, that explanation is provided earlier up the page. However, when I get the AWS STS things rearranged into a more logical home in the install docs, I will make sure it is still defined. That will be a different effort.